gensrc/thrift/PlanNodes.thrift

```diff
@@ -157,8 +157,8 @@ struct TBrokerRangeDesc {
 }
 
 struct TBrokerScanRangeParams {
-    1: required i8 column_separator;
-    2: required i8 line_delimiter;
+    1: required byte column_separator;
```

Comment:
Why using byte instead of i8 here?

Reply:
i got a compile error before when use i8 type.
i will try again in the next commit.